pkgs/tools/networking/modem-manager/default.nix

```diff
@@ -24,6 +24,7 @@ stdenv.mkDerivation rec {
     "--localstatedir=/var"
     "--with-systemd-suspend-resume"
     "--with-systemd-journal"
+    "--with-polkit=permissive"
```

Comment:
I am still not sure this is best practice. Though Fedora disables it completely: https://src.fedoraproject.org/rpms/ModemManager/blob/64f08c48384af5be55b8f9d668ca03613ced3780/f/ModemManager.spec#_106

Reply:
I thought permissive made the most sense with what this PR does.
Other distros that did this included Arch Linux and Void Linux.